### DISCUSSION TITLE
Discover Packages Search UI Question

### DISCUSSION THREAD

**Discussion by SimonZerafa** [Devolutions/UniGetUI#1776] [General]
Hi,

When searching for a package on the Discover Packages Tab, the search indicator (on my system it's a blue line under the toolbar but above the Column which moves from Left to Right) never stops, even when there are clearly no more items to be returned from the search.

Should this search indicator not stop once the search is completed?

If so is there an option in the Settings to do this or is there some other reason why this Search indication never appears to stop?

Kind Regards

Simon Zerafa

**Comment by Spacellary** [comment]
`v2.2.0 - Windows 11`

The indicator does stop here, once the search seems completed.

**Comment by SimonZerafa** [comment]
Hi,

I reset the preferences to default and then performed a search, and it's been going for over 45 minutes with no signs of stopping 🤔

Kind Regards

Simon

**Reply by marticliment** [reply]
Hello Simon,

Which Package Managers do you have enabled?

You can check this from WingetUI Settings

**Reply by Spacellary** [reply]
I'll add that for reference in my test I had **all** package managers enabled.
